TypeScript/permChecker.ts

```diff
@@ -0,0 +1,45 @@
+// @ts-nocheck
+/**
+ * @author: @Nͥeͣoͫʸᵒᵘʳ ˢᶦˢᵗᵉʳ

```

Comment:
If its fine, could you use regular ascii text? This could be a wildcard for some ides and how theyre rendered